src/transformers/onnx/__init__.py

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .config import OnnxConfig, OnnxConfigWithPast, DEFAULT_BERT_OPTIMIZER_FEATURES, EXTERNAL_DATA_FORMAT_SIZE_LIMIT
-from .convert import convert_pytorch
```

Comment:
If we put this in the import, we'll need to have the appropriate `if is_onnxruntime_available()` checks setup. Let me know if you'd like that or if removing it from the init is okay.